microshift_storage/expanding-persistent-volumes-microshift.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="expanding-persistent-volumes-microshift"]
+= Expanding persistent volumes for MicroShift
+include::_attributes/common-attributes.adoc[]
+:context: expanding-persistent-volumes-microshift
+
+toc::[]
+
+include::modules/storage-expanding-add-volume-expansion.adoc[leveloffset=+1]
+
+include::modules/storage-expanding-csi-volumes.adoc[leveloffset=+1]
+
+:FeatureName: Expanding CSI volumes
```

Comment:
This seems really weird without a matching technology-preview.adoc file included. I don't even see it being used in the following module or anything. 

I looked at the history real quick from the assembly you copied this from and I think this is a leftover from #45925 that it didn't get removed. I think you can safely remove it. And @lpettyjo you might want to remove it from the expanding-persistent-volumes.adoc file in a separate PR if you don't mind?